src/models/CommonModel.ts

```diff
@@ -133,11 +133,10 @@ export class CommonModel extends CommonSchema<CommonModel> {
     CommonModel.mergeTypes(mergeTo, mergeFrom);
 
     if (mergeFrom.enum !== undefined) {
-      if (mergeTo.enum === undefined) {
-        mergeTo.enum = mergeFrom.enum;
-      } else {
-        mergeTo.enum = [...mergeTo.enum, ...mergeFrom.enum];
-      }
+      mergeTo.enum = [... new Set([...(mergeTo.enum || []), ...mergeFrom.enum])];
+    }
+    if (mergeFrom.required !== undefined) {
```

Comment:
I wrote simplification of required, but forgot about that 😅

Reply:
Same 😄